### DISCUSSION TITLE
generating mostly black images with sd checkpoints (Mac M3)

### DISCUSSION THREAD

**Discussion by medienassi** [lllyasviel/stable-diffusion-webui-forge#1438] [Q&A]
Hey there, I´ve tried to use forge at my Mac, but **mostly**, not very time, it will end up with a final black image. 
Tried _some_ commandline arguments for ram or no-half etc but to be honest I´m not sure what they do at the Mac hardware.

As example: only prompt: "cat", no negative

Steps: 20, Sampler: DPM++ 2M SDE, Schedule type: Exponential, CFG scale: 7, Seed: 2058813708, Size: 512x640, Model hash: d68f16f125, Model: realisticVisionV51_v51VAE, Version: f2.0.1v1.10.1-previous-399-g852e8856

first: I´ve got an image, went to black at the last steps

------------------------------------------------------

Steps: 25, Sampler: DPM++ 2M SDE, Schedule type: Exponential, CFG scale: 7, Seed: 1379392138, Size: 512x640, Model hash: d68f16f125, Model: realisticVisionV51_v51VAE, Version: f2.0.1v1.10.1-previous-399-g852e8856

second try: final image as it should be

-----------------------------------------------------

Steps: 40, Sampler: DPM++ 2M SDE, Schedule type: Exponential, CFG scale: 7, Seed: 2195359167, Size: 512x640, Model hash: d68f16f125, Model: realisticVisionV51_v51VAE, Denoising strength: 0.7, Hires upscale: 1.5, Hires steps: 25, Hires upscaler: 4x_foolhardy_Remacri, Version: f2.0.1v1.10.1-previous-399-g852e8856

ends in black image OR sometimes mostly black but sometimes with a tiny cat element in there...

I´m a little bit confused, does anyone have so tipps, please?

**Comment by calberts** [comment]
The same here; it's related to the last version. When I installed an old version some months ago, this didn't happen.  (restore  backup)

**Comment by lost-in-emotions** [comment]
i have also this problems. i see my rendered image and at 100% it went full blue, full brown or something.

**so i need to install an older version of forge?**

**Comment by ThoughtCriminal1** [comment]
I'm late to the party, but is there a fix for this? Describes my current problem exactly.
